git add .

```diff
@@ -0,0 +1,14 @@
+identifier: JSON
```

Comment:
This file is created by mistake. Remove it :smiley:

Reply:
uh oh. I'm sorry, I'll remove it now!

Reply:
Np :smile: